auth/client/src/test/java/org/wildfly/security/auth/client/DefaultSSLContextFromFileWorksAndHasPrecedenceTestCase.java

```diff
@@ -0,0 +1,29 @@
+package org.wildfly.security.auth.client;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import javax.net.ssl.SSLContext;
+import java.security.NoSuchAlgorithmException;
+import java.security.Security;
+
+public class DefaultSSLContextFromFileWorksAndHasPrecedenceTestCase {
+    private static final String CONFIG_FILE = "file:./src/test/resources/org/wildfly/security/auth/client/test-wildfly-config-client-default-sslcontext.xml";
+
+    @Test
+    public void testDefaultSSLContextFromFileWorksAndHasPrecedence() {
+        Security.insertProviderAt(new ElytronClientDefaultSSLContextProvider(CONFIG_FILE), 1);
+        Assert.assertNotNull(Security.getProvider("ElytronClientDefaultSSLContextProvider"));
+        AuthenticationContext.empty().run(() -> {   // provider will not use current auth context but will use authentication context from the file passed to provider. File passed to provider has precedence
+            SSLContext cip = null;
```

Comment:
Just for personal knowledge (but maybe worth a renaming), what does `cip` stand for?

Reply:
@fabiobrz sorry it was a bad name, originally it meant `context in provider` but that was just bad naming. I changed it to `default SSLContext`.

Reply:
Thanks